src/startBody.cpp

```diff
@@ -43,18 +45,18 @@ int main(int argC,char **argV)
 	iconv_t charConverter = iconv_open("UTF-8","UTF-16");
 	ros::Publisher bodyPub = n.advertise<k2_client::BodyArray>(topicName,1);
 	char jsonCharArray[readSkipSize];
+   
 	while(ros::ok())
 	{
-		mySocket.readData();
+		mySocket.readData();        
 		char *jsonCharArrayPtr;
 		char *socketCharArrayPtr;
 		jsonCharArrayPtr = jsonCharArray;
 		socketCharArrayPtr = mySocket.mBuffer;
-		iconv(charConverter,&socketCharArrayPtr,&readSkipSize,&jsonCharArrayPtr,&stringSize);
+        iconv(charConverter,&socketCharArrayPtr,&readSkipSize,&jsonCharArrayPtr,&stringSize);
```

Comment:
I think tabs and spaces got mixed in this file.  Can we convert to spaces-only please?